src/geovista/geodesic.py

```diff
@@ -161,7 +161,7 @@ def __init__(
         ----------
         lons : ArrayLike
             The longitudes (degrees) of the bounding-box, in the half-closed interval
-            ``[-180, 180)``. Note that, longitudes will be wrapped to this interval.
+            ``[-180, 180]``. Note that, longitudes will be wrapped to this interval.
```

Comment:
@HGWright Awesome, thanks for taking the time to make this pull-request!

Unfortunately, this is actually correct (as far as I understand).

It's a **half-closed** interval i.e., `-180 ≤ x < 180` which is expressed as `[180, 180)`, given this [interval notation](https://mathworld.wolfram.com/Half-ClosedInterval.html) i.e., `-180°` is the same as `180°` longitude. So in `geovista` `180°` will always be wrapped and represented as `-180°`.

If you're in agreement, then I'll leave you to close this pull-request, although happy to discuss further 😉

Reply:
@bjlittle Apologies, you are right. I was too hasty and jumped to a conclusion. I will close this PR. Thanks

Reply:
@HGWright In hindsight, it's probably worthwhile highlighting this in the `geovista` documentation for the community (so it's more obvious, rather than buried in a docstring) ... so thanks for pointing this out 🚀